cleverhans_tutorials/mnist_tutorial_deepfool.py

```diff
@@ -0,0 +1,184 @@
+# adapted from mnist_tutorial_cw.py
```

Comment:
@goodfeli : Do you think we should keep reproducing tutorials for each attack? This tutorial is adapted from the CW tutorial, which is itself adapted from the JSMA tutorial. 

Perhaps it would make more sense to write one tutorial and have a flag used to choose the attack algorithm?